scripts/skills/perks/perk_rf_fresh_and_furious.nut

```diff
@@ -83,16 +97,30 @@ this.perk_rf_fresh_and_furious <- ::inherit("scripts/skills/skill", {
 	function onAnySkillExecuted( _skill, _targetTile, _targetEntity, _forFree )
 	{
 		this.m.IsSpent = !this.m.IsUsingFreeSkill;
+
+		if (_skill.getID() == "actives.recover")
+		{
+			this.m.RequiresRecover = false;
+			if (this.isEnabled())
```

Comment:
This condition is redundant and not in line with how that effect actually works.

Currently this effect only becomes disabled at the start of a turn, not at any other points in time. And using Recover always activates it, regardless of any other conditions.